ui/src/components/EntryListItem/EntryListItem.tsx

```diff
@@ -146,17 +148,30 @@ export const EntryItem: React.FC<EntryProps> = ({entry, focusedEntryId, setFocus
             </div>}
             <div className={styles.endpointServiceContainer}>
                 <Summary method={entry.method} summary={entry.summary} updateQuery={updateQuery}/>
-                <div className={styles.service}>
+                <div className={styles.resolvedName}>
+                    <Queryable
+                        query={`src.name == "${entry.src.name}"`}
+                        updateQuery={updateQuery}
+                        displayIconOnMouseOver={true}
+                        style={{marginTop: "-4px", marginRight: "10px"}}
+                    >
+                        <span
+                            title="Source Name"
+                        >
+                            {entry.src.name ? entry.src.name : entry.src.ip}
+                        </span>
+                    </Queryable>
+                    <div style={{marginRight: "10px"}}>{"->"}</div>
                     <Queryable
-                        query={`service == "${entry.service}"`}
+                        query={`dst.name == "${entry.dst.name}"`}
```

Comment:
if I understand correctly when there dest.name does not exists this filter will add "dst.name == undefined " or "dst.name == null" (because when there is no name we are showing ip. but this will not be the filter that will be added... maybe we need to address this difference ?